qp_klp/tests/test_klp.py

```diff
@@ -341,6 +344,18 @@ def test_sequence_processing_pipeline(self):
 
         self.assertEqual(ainfo, exp)
 
+        # verify cmd.log
```

Comment:
I think it will be better to have a multiline string with all the commands that are expected so it's easier to read while reviewing the code; what do you think?

Reply:
I agree! I think that's a good idea. I just added that to the test.